### PR TITLE
 bumped next-version to 1.46.0

### PR DESCRIPTION
**Pull request by XiaotianNetlift**
## PR Type
What kind of change does this PR introduce?
- Build or CI related changes
 - Documentation content changes

## What is the new behavior?
bumped next-version to 1.46.0 from 1.45.0

## PR Checklist
Please check if your PR fulfills the following requirements:
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

## Other information
Internal Issue (If applicable):
https://nventive.visualstudio.com/Umbrella/_workitems/edit/156042

### CODE REVIEW

**Comment by MatFillion** [comment]
We shouldn't have updates to beta in master.

**Comment by jeromelaban** [comment]
Agreed. Don't update packages to the latest beta, latest dev is fine.

**Comment by XiaotianNetlift** [comment]
@MatFillion @jeromelaban I fixed it. Can you review this again?

File: doc/ReleaseNotes/_ReleaseNotes.md [hunk 1]
```diff
@@ -87,6 +84,17 @@
 * [Wasm] Add `PathIcon` support
 * [Wasm] Add UI Testing support through for `Uno.UI.Helpers.Automation.GetDependencyPropertyValue`\
 * Add automated UI testing support for every PR
+* Add support for `Windows.Phone.Devices.Notification.VibrationDevice` API on iOS, Android and WASM
```
Comment (jeromelaban) [inline doc/ReleaseNotes/_ReleaseNotes.md]: Are you going to merge this PR in the beta as well ?

Comment (MatFillion) [reply doc/ReleaseNotes/_ReleaseNotes.md]: yes @jeromelaban  its already on the beta branch : https://github.com/nventive/Uno/commits/release/beta/Batman

The release/beta/Batman branch actually

Comment (XiaotianNetlift) [reply doc/ReleaseNotes/_ReleaseNotes.md]: It is **not** part of current beta. I moved this out of 1.45.0.

**Pull request merged**